canonical-websites/www.ubuntu.com
# 3279 developer menu

## Done

Update developer menu

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Open developer menu
- See that it matches the [copy doc](https://docs.google.com/document/d/1JzZPBydcdyV96Hu4xRpa7VDU_WREr6gb05bL7GlQaYY/edit#)


## Issue / Card

Fixes #3279
